src/gui/IconModels.h

```diff
@@ -41,14 +41,14 @@ class CustomIconModel : public QAbstractListModel
 public:
     explicit CustomIconModel(QObject* parent = nullptr);
 
-    int rowCount(const QModelIndex& parent = QModelIndex()) const override;
-    QVariant data(const QModelIndex& index, int role = Qt::DisplayRole) const override;
-    void setIcons(const QHash<Uuid, QImage>& icons, const QList<Uuid>& iconsOrder);
+    virtual int rowCount(const QModelIndex& parent = QModelIndex()) const Q_DECL_OVERRIDE;
```

Comment:
Great question, I don't recall.  Likely two minor bugs.  I think you're the first person to review the code. Ever. :)